content/about/open-letter-meta-dma/technical-briefing.md

```diff
@@ -0,0 +1,271 @@
+
+### The Consequences: A Fully Closed Messaging Landscape
+
+Facebook was not alone in this strategy, but it played an **instrumental role in locking down messaging communication** before full interoperability could be realized. What makes this situation even more frustrating is that:
+
+- **Facebook Messenger originally supported XMPP**.
+- **WhatsApp has always been built on an XMPP-based server**—its **internal logic remains rooted in XMPP**, yet it operates as a completely closed system.
+- Both used the same open source XMPP server to bootstrap their infrastructure.
+
+As a result, we have **regressed to a fragmented messaging landscape reminiscent of 1999**, before the emergence of federated interoperability. Today, **multiple closed messaging ecosystems compete to dominate the communication space**, including:
+
+- **Facebook Messenger**
+- **WhatsApp**
+- **Instagram private messaging**
+- **Telegram**
+- **WeChat**
+- **LinkedIn Messaging**
+- **Various Google messaging services**
+- And many more
+
+Each of these services **operates in isolation**, forcing users to either create accounts on multiple platforms to stay in touch with all their contacts or rely on a single provider—driving further concentration in the sector. If a user chooses to focus on just one app, they must convince all their contacts to do the same, effectively reinforcing a **global-scale monopoly** in communication.
+
+### Beyond Messaging: The Broader Lock-In of Communication
+
+What is worst, is that this issue is no longer limited to text-based messaging. Modern messengers have expanded to **voice and video calls**, effectively **controlling digital communication as a whole**. If we do not change course, the endgame is clear, as we are stuck in a race toward a **global monopoly on internet communication**, where a handful of gatekeepers could dictate their terms, limiting competition and user freedom.
+
+## The European Commission and the Digital Markets Act
+
+To prevent extreme concentration of power in the messaging sector, the **European Commission introduced the Digital Markets Act (DMA)**. Its goal is to ensure that communication remains **open, fair, and competitive**, preventing a future where global messaging is controlled by a **monopoly, duopoly, or, at best, an oligopoly**. Enforcing interoperability is the only way to allow users to **freely choose their preferred messaging provider** rather than being locked into dominant platforms.
```

Comment:
Slightly tweaked the wording - the DMA is not *only* about messaging.

```suggestion
To prevent extreme concentration of power in the messaging sector, the **European Commission included it firmly within the scope of the Digital Markets Act (DMA)**. Its goal is to ensure that communication remains **open, fair, and competitive**, preventing a future where global messaging is controlled by a **monopoly, duopoly, or, at best, an oligopoly**. Enforcing interoperability is the only way to allow users to **freely choose their preferred messaging provider** rather than being locked into dominant platforms.
```